core/server/data/migrations/versions/5.0/2022-04-28-03-26-remove-author-id-column-from-posts-table.js

```diff
@@ -0,0 +1,7 @@
+const {createDropColumnMigration} = require('../../utils');
+
+module.exports = createDropColumnMigration('posts', 'author_id', {});
```

Comment:
We can't use the utility on the `posts` table because it doesn't include `algorithm=copy`, which is a performance optimization needed for large tables